css/properties.json

```diff
@@ -4220,6 +4220,50 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain"
   },
+  "contain-intrinsic-size": {
+    "syntax": "[ none | <length> | auto <length> ]{1,2}",
+    "media": "visual",
+    "inherited": false,
+    "animationType": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "percentages": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "groups": [
+      "CSS Containment"
+    ],
+    "initial": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "appliesto": "allElements",
```

Comment:
I think this is wrong? Wouldn't this be the same as for the longhand:

> elements with [size containment](https://w3c.github.io/csswg-drafts/css-contain-2/#size-containment)

?

Reply:
Yes. Thanks
```suggestion
    "appliesto": "elementsForWhichSizeContainmentCanApply",
```